pkg/backup/item_backupper_test.go

```diff
@@ -455,7 +458,7 @@ func TestTakePVSnapshot(t *testing.T) {
 			name:            "unsupported PV source type",
 			snapshotEnabled: true,
 			pv:              `{"apiVersion": "v1", "kind": "PersistentVolume", "metadata": {"name": "mypv"}, "spec": {"unsupportedPVSource": {}}}`,
-			expectError:     false,
+			expectError:     true,
```

Comment:
Why did this change from false to true? Shouldn't it still not return an error since it doesn't contain a supported PV source?

Reply:
You're right, thanks.